src/features/messagePruner/pruneMessagesCommand.ts

```diff
@@ -9,10 +9,10 @@ import {
   TextChannel,
 } from 'discord.js'
 
-import { defineCommandHandler } from '@/types/defineCommandHandler'
+import { CommandHandlerConfig } from '@/types/CommandHandlerConfig'
 import { isInArray } from '@/utils/typeGuards'
 
-export default defineCommandHandler({
+export const pruneMessagesCommand: CommandHandlerConfig = {
```

Comment:
Why it isn't using define pattern anymore 🤔

Reply:
because in part 3, defineCommandHandler no longer returns a `CommandHandlerConfig` but returns a `Plugin`.

i will rename to `defineCommand` instead krub.